src/tests/Presenters/UserSettingsFake.cs

```diff
@@ -28,6 +28,9 @@
 
 namespace NUnit.Gui.Presenters
 {
+    // TODO: With latest changes, this is no longer used.
+    // Keeping it for a while, as we may end up wanting
+    // to write more tests that use it.
     class UserSettingsFake : ISettings
```

Comment:
Maybe consider marking this as obsolete with the same message, then either delete it or flip the flag to make it cause a compile error.

Reply:
Since it's in our tests, I see no harm in it for now. This is part of a series of changes to the same code so I'll be coming back here soon.